### DISCUSSION TITLE
The deployed version shows an entirely different design from what I made 😔

### DISCUSSION THREAD

**Discussion by lithika-damnod** [vercel/community#990] [Help] [answered]
Hi 🙌 
I made some changes to the web page ( react ) and pushed changes to the repo.. so vercel could deploy it automatically ...
But something crazy happened this time, 
It was showing an entirely different web page than it was looking in my local environment.  ( it didn't even have loaded up my new images properly ) 

**Local Environment** 
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/61654812/198054491-a41c7389-7ea9-41ec-8cef-11198860b7fa.png">
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/61654812/198054521-b4712ad5-b7b2-44dc-8455-8525c7ad7047.png">

**Vercel ( After Deploying )** 
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/61654812/198054742-04a71bb9-8d40-4aef-93ce-f3872d72100d.png">
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/61654812/198054782-9906b8b9-9dd1-4d4b-b23e-dca542028925.png">

_link to the webpage:_  https://sampathmotors.vercel.app/

**Comment by amyegan** [comment, marked as answer]
Hi @lithika-damnod! Thanks for sharing these details and a link to the deployment. I'm seeing a `div` with 100% height and width. If I remove this `div` I am able to see your site as shown in your first example image. 
```
<div class="banner-wallpaper" style="opacity: 1; transform: scale(1);"></div>
```
```
.banner-container .banner-wallpaper {
    background-color: purple;
    background-position: 50%;
    background-size: cover;
    height: 100%;
    left: 0;
    position: absolute;
    top: 0;
    width: 100%;
    z-index: 2;
}
```

You might have some cached styles on your local computer, or a difference in local build or browser settings, which may cause you to see something different when running in on localhost.

**Reply by lithika-damnod** [reply]
Thanks for noting this! @amyegan  🙌
After seeing your reply I headed to my CSS file and checked whether I've forgotten those styles.
But all seemed to be fine, I had done the styling correctly.. 
But what I noticed is that vercel is not properly loading CSS just only for that component.
so I tried moving all the CSS into inline CSS .. which afterward made it work fine.. ( I still don't understand what really happened ) 
However, thank you very very much for the help .. 🙏
